### DISCUSSION TITLE
Upstream from Prestige Deployment?

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#420] [Q&A] [answered]
Also, sync up August 1.

**Comment by hollowaykeanho** [comment, marked as answer]
I upstreamed the project management layout already. Yes, please do. We will do 1 more minor release (`v.1.3.0`) with all the discovery before building nim.

**Comment by corygalyna** [comment]
Ok. Anything else?

**Comment by hollowaykeanho** [comment]
Yes, please file them on your side. My system will be down for a while due to Debian maintenances. I will try to restore everything up by Wednesday.

**Comment by corygalyna** [comment]
Got it.
